a10_octavia/controller/worker/tasks/decorators.py

```diff
@@ -14,14 +14,33 @@
 
 
 import acos_client
+from acos_client.errors import ACOSException
+from acos_client.errors import Exists
```

Comment:
One of the Openstack coding guidelines is to not import just a single class.

> Imports¶
Do not import objects, only modules; exceptions are:
imports from migrate package
imports from sqlalchemy package
function imports from i18n module

So instead reformat imports as follows

```python
from acos_client import errors as acos_errors (or a better name)
try:
    import http.client as http_client
except ImportError:
    import httplib as http_client
...
from requests import exceptions as req_exceptions (or a better name)
```

Reply:
@hthompson-a10  addressed this in PR https://github.com/a10networks/a10-octavia/pull/70